guides/common/modules/proc_salt-using-content-management.adoc

```diff
@@ -0,0 +1,23 @@
+[id="salt_guide_using_content_management_{context}"]
+= Using {Project}s Content Management
+
+This section provides best practice recommendations on how to provide managed hosts with the required Salt Minion client software using {Project}.
+
+.Procedure
+. Create and synchronize the required products and repositories.
+.. Create a product called `Salt`.
+.. Create a repository within the `Salt` product for each operating system supported by {Project} that you want to install the Salt Minion client software on.
++
+Add the operating system to the name of the repository, for example `Salt for Ubuntu 20.04`.
++
+You can find the _upstream URL_ for the Salt packages on the https://repo.saltproject.io/[official Salt package repository].
+The URL depends on both the Salt version and the operating system, for example `\https://repo.saltproject.io/py3/ubuntu/20.04/amd64/3003/`.
+.. Synchronize the previously created products.
```

Comment:
I see no value in the substeps and would prefer everything on the same level.

Reply:
Removed the substeps.